lib/matplotlib/axes/_axes.py

```diff
@@ -3585,6 +3593,32 @@ def scatter(self, x, y, s=20, c='b', marker='o', cmap=None, norm=None,
         if not self._hold:
             self.cla()
 
+        # Process **kwargs to handle aliases, conflicts with explicit kwargs:
+
+        facecolors = None
+        ec = kwargs.pop('edgecolor', None)
+        if ec is not None:
+            edgecolors = ec
+        fc = kwargs.pop('facecolor', None)
+        if fc is not None:
+            facecolors = fc
+        fc = kwargs.pop('facecolors', None)
+        if fc is not None:
+            facecolors = fc
+        # 'color' should be deprecated in scatter, or clearly defined;
+        # since it isn't, I am giving it low priority.
+        co = kwargs.pop('color', None)
+        if co is not None:
+            if edgecolors is None:
+                edgecolors = co
+            if facecolors is None:
+                facecolors = co
```

Comment:
Coming in late to the discussion but I have just spent a few minutes trying to determine what I did wrong on my scatter graphs...

I thought ``c`` stood as an acronym for ``color`` as ``fc`` stands for ``facecolor`` etcetera... however this does not apply here, specifying ``color='green'`` instead of ``c='green'`` means you set the edge-colour to green instead of the face-colour...

Should I open an issue/PR about this?  Or can people live with this inconsistency?